extensions/amp-analytics/0.1/vendors/webengage.js

```diff
@@ -33,6 +36,22 @@ const WEBENGAGE_CONFIG = jsonLiteral({
       'request': 'wePageview',
     },
   },
+  'linkers': {
+    '_we_linker': {
+      'destinationDomains': ['*'],
+      'enabled': true,
+      'ids': {
+        'we_luid': '${clientId}',
```

Comment:
I would suggest changing this too, due to the confusion in may create in the future.